src/core/reducers/errors.js

```diff
@@ -12,14 +12,21 @@ import { gettext } from 'core/utils';
  * http://addons-server.readthedocs.io/en/latest/topics/api/overview.html#responses
  */
 function getMessagesFromError(error) {
-  let messages = [gettext('An unexpected error occurred')];
+  let data = {
+    code: undefined,
```

Comment:
I feel like this should be its own code (`UNKNOWN_ERROR`) and that way the component can use real `i18n.gettext` for it.

I'm guessing this path should never happen but I think we might as well handle it correctly.